app/scripts/views/sign_up.js

```diff
@@ -34,7 +34,7 @@ define(function (require, exports, module) {
     } else if (! password) {
       return 'input[type=password]';
     } else {
-      return '#age';
```

Comment:
We'll still want to go to #age if there is both a password and a vpassword

Reply:
@shane-tomlinson  but how can there be vpassword, user has to manually type it while signing up or it should be prefilled like password field ?

Reply:
@farhan787 If you wanted to do this check you would probably need to update the function [here](https://github.com/mozilla/fxa-content-server/blob/881efbc9a626a167aaa4d76291ef0cdd3db99d78/app/scripts/views/sign_up.js#L91) to find the "#vpassword" element and then pass it along to this function where you can select appropriate element to focus.

Reply:
@vbudhram thanks for help, I was doing that but I was confused about why do we need to check for retype password(vpassword) field, but now @shane-tomlinson cleared my doubt.